homeassistant/components/binary_sensor/__init__.py

```diff
@@ -47,3 +57,14 @@ def state(self):
     def friendly_state(self):
         """ Returns the friendly state of the binary sensor. """
         return None
+
+    @property
+    def sensor_class(self):
+        """ Returns the class of this sensor, from SENSOR_CASSES. """
+        return 'digital'
```

Comment:
Wonder if we should make `None` default and not add it as an attribute when not set. The UI will do the default behavior.